### DISCUSSION TITLE
[BUG] Actions - Renaming directory containing sub-directories deletes folders with spaces

### DISCUSSION THREAD

**Discussion by TheXRMonk** [community/community#27008] [Actions]
<p>I’m running this action which works fine except the fact that any folders with a space in their name inside the Samples folder will not be committed. </p>
<pre><code>if [[-d "Samples"]]; then
          git rm -f -r Samples~
          git mv Samples Samples~
          rm -f Samples.meta
          git config --global user.name 'github-bot'
          git config --global user.email 'github-bot@users.noreply.github.com'
          git commit -am "fix: Samples =&gt; Samples~"
        fi
</code></pre>
<p>Very weird to me.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/virsabimunk">@virsabimunk</a> ,</p>
<p>After moving to " <strong>Samples~</strong>" directory, you can run the command " <strong>ls -R</strong>" to check if all the files in " <strong>Samples</strong>" directory and its sub-directories have been moved successfully. You also can try the <strong>mv</strong> command (not <strong>git mv</strong> ) to move the files.</p>
<p>And before executing <strong>git commit</strong> command, you also can try the <a href="https://git-scm.com/docs/git-add" rel="nofollow noopener"><strong>git add</strong></a> command.<br>
For example:</p>
<pre><code>git add Samples~
git commit -am "fix: Samples =&gt; Samples~"
</code></pre>
<p>If the problem still exists, please share your repository with us, so that we can check more detailed configurations of your workflow in the repository to analyze the root cause.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/virsabimunk">@virsabimunk</a> ,</p>
<p>How are things going? Is my above suggestion helpful to you?</p>
<p>Please try it, and any progress, feel free to tell me.</p>

**Comment by TheXRMonk** [comment]
<p>I’m sorry, it was another third party action that was the culprit.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/virsabimunk">@virsabimunk</a> ,</p>
<p>Glad that you have found the root cause and solved the problem.</p>
<p>Have a nice day.</p>

**Comment by Dr-AMH** [comment]
Let me know how to edit the name wordRiddles_HCG using GUI not commandLine please.

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/77119521/187687003-beb28199-2265-45a7-bf66-d9f0843defd8.png">
